scripts/copyComponentExperiencesYaml.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
```

Comment:
Nit: let's name this a bit more generically, e.g. `copyResources.js`. Opens us up to copying more resources in the future, without renaming the file.